### DISCUSSION TITLE
OUR TEAM`S DISCUSSION MOVED HERE (message to ZURI)

### DISCUSSION THREAD

**Discussion by Clin-Tech** [zuri-training/Fetch_Metadata_Team82#44] [General]
**We would love to make this issue known to the Zuri mentors or coordinators in charge of this team that we have moved our discussion and project tab from the Github Team section to this Github Repository section because after submitting the link in the first presentation we got the feedback that the link was displaying "404 This Is Not The Web Page You Are Looking For."**

This was so disheartening that even after trying to call attention of the help and supervisor to it , we couldn`t get it resolved so we resorted to starting a new discussion and project in this Repository section and not the Team section.

**PICTORIAL EVIDENCES ATTACHED BELOW TO BACKUP OUR STATEMENT.**

![Screenshot (23)](https://user-images.githubusercontent.com/105053533/183754437-eed3d774-168a-41db-bf58-f6e5b0dbc6c0.png)

![Image from iOS (1)](https://user-images.githubusercontent.com/105053533/183755968-6a3ac4b2-a388-4631-9adb-4a2571f1ce59.jpg)
![Screenshot (24)](https://user-images.githubusercontent.co
![Screenshot (23)](https://user-images.githubusercontent.com/105053533/183756664-419b118a-e249-4599-933b-aef7a8a7c4cd.png)
m/105053
![Screenshot (21)](https://user-images.githubusercontent.com/105053533/183756637-5f95cffc-4375-4c6d-bac8-d3419ebc6cff.png)
533/183756582-3dfe04a6-396a-484c-aa07-6086f243c177.png)

**Comment by FitsumMehari** [comment]
Yes, We had a lot of discussions going long before the first presentation, but for some reason we go 0/4. I hope Zuri can understand that we were and still are active on git and github.

**Comment by Oladejo-ayomide** [comment]
Hopefully zuri will understand our plea. We have always and will be active on github.

**Comment by Osayomwanbo** [comment]
Zuri please try to understand us... We have always    been active and we are still active on GitHub

**Comment by Prissy-Ama** [comment]
Due to this 404 issue we got a rating of 0/4 from our first presentation. We tried all we could after learning that our page wasn't publicly accessible but couldn't solve the issue. I hope the Zuri team understands and are more lenient.

**Comment by KosiNina** [comment]
Please, do understand, we have been very active here.

**Comment by iamelovia** [comment]
We have been active since the GitHub team page was created. Please bear with us. Thank you

**Comment by BunmiFash** [comment]
True! 4 marks gone like that.  Please do consider us as we had no choice.

**Reply by Somlivia** [reply]
We have been active here but due to the issue we had, we lost 4 marks.

Zuri team, please bear with us

**Comment by Adesola-Adepoju** [comment]
Please,we have been active here.
